storm-server/src/main/java/org/apache/storm/scheduler/TopologyDetails.java

```diff
@@ -84,6 +89,8 @@ public TopologyDetails(String topologyId, Map<String, Object> topologyConf, Stor
         initConfigs();
         this.launchTime = launchTime;
         this.topoName = (String) topologyConf.get(Config.TOPOLOGY_NAME);
+        this.distributeSysCompEvenly
```

Comment:
This property is more useful in Scheduler code where is can be used to sort executors in a manner conducive to even distribution.